velox/exec/TaskStructs.h

```diff
@@ -111,11 +111,18 @@ struct SplitGroupState {
   /// e.g. Limit.
   uint32_t numFinishedOutputDrivers{0};
 
+  // True if the state contains structures used for connecting ungrouped
+  // execution pipeline with grouped excution pipeline. In that case we don't
+  // want to clean up some of these structures.
+  bool crossExecutionMode{false};
```

Comment:
@mbasmanova
Yes, I didn't like that cross myself. :)
The reason I used cross is to indicate that the bridge connects two different executions.
Because in a mixed execution task you can have bridges that don't cross execution boundaries.
"mixed" would be all right here, I believe.